inventories/seapath_basic_cluster_definition_example.yml

```diff
@@ -5,15 +5,16 @@ all:
             children:
                 hypervisors:
```

Comment:
Your commit title is to general to understand quickly what it does. 
A good practice is <FILE NAME>: <COMMIT PURPOSE> (refactor documentation, etc)